jest.config.js

```diff
@@ -0,0 +1,16 @@
+/** @type {import('ts-jest').JestConfigWithTsJest} */
+export default {
+  preset: 'ts-jest',
+  testEnvironment: 'jsdom',
+  setupFilesAfterEnv: ['<rootDir>/src/setupTests.ts'],
+  moduleNameMapper: {
+    '\\.css$': 'identity-obj-proxy',
+    '\\.(jpg|jpeg|png|gif)$': '<rootDir>/src/__mocks__/fileMock.ts',
+    '^components/(.*)$': '<rootDir>/src/components/$1',
+    '^constants/(.*)$$': '<rootDir>/src/constants/$1',
+    '^context/(.*)$': '<rootDir>/src/context/Context',
+    '^helpers/(.*)$': '<rootDir>/src/helpers/$1',
+    '^assets/(.*)$': '<rootDir>/src/assets/$1',
+    '^apis/(.*)$': '<rootDir>/src/apis/$1',
```

Comment:
You can always use the following notation:
```js
{
  moduleNameMapper: {
    '^@/(.*)$': '<rootDir>/src/$1',
  }
}
```
This change would require additional change in tsconfig + updating the imports in the codebase. You can create a new refactoring issue and work on that once you have time.

What do you get with the proposed approach? Adding an alias to ts config and jest config is required just once. Any change in folder structure (adding a new folder in the src root) will not require additional change in these configurations.

Reply:
Thanks for this suggestion!
Because it is a one-time effort and it will help me in the future, I added this in the [last commit](https://github.com/zuko4celik/movies-app-new/pull/24/commits/2c9a79b44c2ed1a1bfa637b9d8af0a4b3fe064be).